src/components/Dropdown/asyncDropdown/AsyncSearchTags.tsx

```diff
@@ -79,7 +79,7 @@ const AsyncSearchTags = ({
 
   const createNewTag = (input: string) => {
     if (input) {
-      const newTags = input.split(',');
+      const newTags = [...new Set(input.split(','))];
```

Comment:
Det er litt merkelig behaviour om man legger inn `en, en, en` som tags da den tar med spaces. Burde vi kanskje trimme whitespace etter splitting?

Reply:
Etter diskusjon på fredag så fjerner eg støtten for innlesing av kommaseparert liste for nøkkelord, så kjem en oppdatert versjon her snart.